src/routes/MagicianRoutes.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Please kindly do not disable linters, it's better to fix linters it helps to write a clean and good codebase 😉